server/service/src/sync/translations/invoice.rs

```diff
@@ -94,8 +94,7 @@ pub struct LegacyTransactRow {
     #[serde(rename = "prescriber_ID")]
     pub clinician_id: Option<String>,
     #[serde(rename = "currency_ID")]
-    #[serde(deserialize_with = "empty_str_as_option_string")]
-    pub currency_id: Option<String>,
```

Comment:
@Chris-Petty is `currency_id` guaranteed to be not NULL in OG? 4D table says it can be NULL...